src/views/Dashboard/components/DashboardCards/AuthorizedStudies/index.tsx

```diff
@@ -23,16 +24,34 @@ import styles from './index.module.scss';
 
 const AuthorizedStudies = ({ id, className = '' }: DashboardCardProps) => {
   const dispatch = useDispatch();
-  const { loadingStudiesForFences, fenceStudiesAcls, isConnected, hasErrors, connectionLoading } =
-    useFenceStudies();
+  const {
+    loadingStudiesForFences,
+    fenceStudiesAcls,
+    isConnected: isGen3Connected,
+    hasErrors: hasGen3Errors,
+    connectionLoading: connectionGen3Loading,
+  } = useFenceStudies(FENCE_NAMES.gen3);
+
+  const {
+    //Shared: loadingStudiesForFences,
+    //Shared: fenceStudiesAcls,
+    isConnected: isDcfConnected,
+    hasErrors: hasDcfErrors,
+    connectionLoading: connectionDcfLoading,
+  } = useFenceStudies(FENCE_NAMES.dcf);
+
   const fenceStudiesLoading = loadingStudiesForFences.length > 0;
 
+  const isOneFenceAtLeastConnected = isGen3Connected || isDcfConnected;
+  const hasAtLeastOneFenceWithErrors = hasGen3Errors || hasDcfErrors;
```

Comment:
wondering if it can be a problem - for, one error will compromised the whole component .From my local env it was OK though.